pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -632,12 +633,16 @@ func (p *PriorityQueue) MoveAllToActiveQueue() {
 		} else {
```

Comment:
We should remove this else and call continue in the above if:

https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow